server/resp/src/main/java/org/infinispan/server/resp/RespServer.java

```diff
@@ -40,7 +41,15 @@ protected void startInternal() {
       if (!cacheManager.getCacheManagerConfiguration().features().isAvailable(RESP_SERVER_FEATURE)) {
          throw CONFIG.featureDisabled(RESP_SERVER_FEATURE);
       }
-      if (cacheManager.getCacheConfiguration(configuration.defaultCacheName()) == null) {
+      String defaultRespCacheName = configuration.defaultCacheName();
+      Configuration respCacheConfig = cacheManager.getCacheConfiguration(defaultRespCacheName);
+      if (respCacheConfig != null) {
+         ExpirationConfiguration expConfig = respCacheConfig.expiration();
+         if (expConfig.lifespan() >= 0 || expConfig.maxIdle() >= 0)
+            throw log.invalidExpiration(configuration.defaultCacheName());
+         // If the respCache configuration is explicitly defined, start the cache
```

Comment:
@wburns I added this comment, I thought this was the comment you requested

Reply:
That isn't what I was referring to, but the comment isn't as valuable now that you moved it to use `startCaches` which is a lot more explicit imo. But I would say the comment is better for the other case, saying we only start the cache on demand if it wasn't explicitly configured.

But I am fine this way.